website/docs/r/virtual_machine_scale_set_extension.html.markdown

```diff
@@ -28,6 +28,13 @@ resource "azurerm_linux_virtual_machine_scale_set" "example" {
   admin_username      = "adminuser"
   instances           = 1
 
+  source_image_reference {
+    publisher = "Canonical"
+    offer     = "UbuntuServer"
+    sku       = "16.04-LTS"
```

Comment:
This version is EOL by Canonical, is it worth bumping to `20.04-LTS` ?

Reply:
sure, I've updated the versions to 20.04